src/firefly/java/edu/caltech/ipac/firefly/data/DownloadRequest.java

```diff
@@ -74,7 +74,7 @@ public void setFilePrefix(String filePrefix) {
 
     public String getDataSource() { return getParam(DATA_SOURCE); }
 
-    public String getTitle() { return getParam(TITLE); }
+    public String getTitle() { return getParam(BASE_FILE_NAME); }
```

Comment:
@loitly I need your feedback on this change. Earlier we had two fields (title and file name) in the download dialog (with `fieldKey` set to `title` and `BaseFileName` respectively). 

Now, these are both used on the server side (`BaseFileName` as the file name) and `title` is used here. 

After merging the title and file name fields into just one 'title' field on the UI, I also changed the `fieldKey` to `BaseFileName`, because the title will also be used as the file name now. 

So with that in mind, does this change look okay? Any alternate suggestions?